src/python/dxpy/bindings/dxfile.py

```diff
@@ -112,24 +120,29 @@ def _ensure_http_threadpool(cls):
         if cls._http_threadpool is None:
             cls._http_threadpool = dxpy.utils.get_futures_threadpool(max_workers=cls._http_threadpool_size)
 
+    @classmethod
+    def _set_file_limits(cls, file_limits):
+        cls._buffer_size = file_limits['minimumPartSize']
```

Comment:
These should be instance variables so that clients can use file bindings against files in multiple clouds in the same process.